### PR TITLE
enhanced_login_fix#3

### PR DESCRIPTION
**Pull request by OMIIIEE**
enhanced login page 
image

### CODE REVIEW

**Comment by TzivyaFireman** [comment]
The design is really interesting and beautiful!
However, there's an issue with the user authentication: currently, the user credentials are not being properly validated. This means that even with incorrect username and password, the system redirects the user to the next page. The authorization process is not working as expected, so I would appreciate it if you could fix this to ensure proper user validation before proceeding.

File: wwwroot/bg.jpg
[the remark this answers is not in the document]
Comment (OMIIIEE) [reply wwwroot/bg.jpg:1]: yes

**Comment by OMIIIEE** [comment]
updated index.js and now login is not done with incorrect name and password
image

**Review by TzivyaFireman — APPROVED** [review]
Thank you for designing the page and paying attention to the small details,
it truly makes a difference!

**Pull request merged**